tla-io/src/main/scala/at/forsyte/apalache/io/lir/ItfCounterexampleWriter.scala

```diff
@@ -49,11 +49,20 @@ class ItfCounterexampleWriter(writer: PrintWriter) extends CounterexampleWriter
     val mappedStates = state0 :: states.drop(2).map(_._2)
     // construct the root JSON object
     val rootMap: mutable.LinkedHashMap[String, ujson.Value] = mutable.LinkedHashMap()
+
+    val metaInformation: List[(String, ujson.Value)] =
+      List[(String, ujson.Value)](
+          "format-description" -> "https://apalache.informal.systems/docs/adr/015adr-trace.html",
```

Comment:
I think this can be made a `Map`?

```suggestion
    val metaInformation: Map[String, ujson.Value] =
      Map[String, ujson.Value](
          "format-description" -> "https://apalache.informal.systems/docs/adr/015adr-trace.html",
```

Reply:
Or rather a mutable `scala.collection.mutable.Map`?